docs/supported_models.md

```diff
@@ -271,6 +271,17 @@ Neural Speed supports the following models:
     <td> </td>
     <td> </td>
     <td>Latest</td>
+  </tr>
+    <tr>
+    <td><a href="https://huggingface.co/google/gemma-2b-it" target="_blank" rel="noopener noreferrer">gemma-2b-it </a>,
+    <a href="https://huggingface.co/google/gemma-7b" target="_blank" rel="noopener noreferrer">gemma-7b</a></td>
+    <td>✅</td>
+    <td> </td>
+    <td> </td>
+    <td>✅</td>
```

Comment:
and do we support gguf of this model ? 
https://huggingface.co/lmstudio-ai/gemma-2b-it-GGUF

Reply:
support gguf model convert from llama.cpp